.gitignore

```diff
@@ -31,6 +31,7 @@
 
 # Ignore built binaries
 /ocm-example-service
+/rh-trex
```

Comment:
built binaries don't usually have a hyphen.

We would need to change the directory from `cmd/rh-trex` to `cmd/trex`.